spec/features/proposal/collection_draft_proposals/collection_draft_proposal_submit_spec.rb

```diff
@@ -0,0 +1,107 @@
+describe 'Collection Draft Proposal Submit', js: true do
+  before do
+    login
+  end
+
+  context 'when submitting a validated proposal' do
+    before do
+      set_as_proposal_mode_mmt(with_required_acl: true)
+      @collection_draft_proposal = create(:full_collection_draft_proposal)
+      visit collection_draft_proposal_path(@collection_draft_proposal)
+    end
+
+    context 'when the proposal is submitted' do
+      before do
+        click_on 'Submit for Review'
+      end
+
+      it 'submits a proposal' do
+        click_on 'Yes'
+        expect(page).to have_content('Collection Draft Proposal Submitted Successfully')
+        expect(page).to have_no_link('Temporal Information')
+      end
+
+      it 'does not navigate away when cancelling' do
+        click_on 'No'
+        expect(page).to have_link('Temporal Information')
+      end
+    end
+
+    context 'when the submission fails' do
+      before do
+        # After loading the page, manipulate the state of the proposal so that
+        # submit will fail in order to execute the else code in the controller.
+        proposal = CollectionDraftProposal.first
+        proposal.proposal_status = 'submitted'
+        proposal.save
+        click_on 'Submit for Review'
+        click_on 'Yes'
+      end
+
+      it 'provides an error message' do
+        expect(page).to have_content('Collection Draft Proposal was not submitted successfully')
+        expect(page).to have_link('Rescind Draft Submission')
```

Comment:
If it was not submitted successfully, should rescind be an option?

Reply:
Manually setting the proposal status to submitted means that rescind should be available.  This can be changed to be more intuitive when more states are added; e.g. manually set it to done instead of the intended destination.